scripts/check-public-api.ts

```diff
@@ -85,11 +85,18 @@ function compareApisAndLog(
   verbose: boolean
 ): boolean {
   let setsAreEqual = true;
+  const schemaPathRegex = /.*\/client\/[^/]+\/schema\/[^/]+\.ts$/;
 
   allExportedTypes.forEach(exportedType => {
     if (
       !allExportedIndex.find(nameInIndex => exportedType.name === nameInIndex)
     ) {
+      if (exportedType.path.split(sep).join(posix.sep).match(schemaPathRegex)) {
```

Comment:
@marikaner @shibeshduw  I adjusted the check public api script here to not complaint if I have a type specifically in `./schema` that is not exported from index.ts. Logs a warning for those cases. We can remove it once we fix our generation or del unwanted types. Just FYI

Reply:
I linked this to AI/gen-ai-hub-sdk-js-backlog#115.